app/src/main/java/com/example/mymovie/ui/detail/DetailFragment.java

```diff
@@ -0,0 +1,207 @@
+package com.example.mymovie.ui.detail;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.MotionEvent;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.CheckBox;
+import android.widget.CompoundButton;
+import android.widget.ImageView;
+import android.widget.ListView;
+import android.widget.RatingBar;
+import android.widget.ScrollView;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+
+import com.example.mymovie.CommendShowActivity;
+import com.example.mymovie.CommendWriteActivity;
+import com.example.mymovie.MainActivity;
+import com.example.mymovie.R;
+import com.example.mymovie.ReviewAdapter;
+import com.example.mymovie.ReviewitemData;
+
+import java.util.ArrayList;
+
+import static com.example.mymovie.CommendWriteActivity.COMMEND;
+import static com.example.mymovie.CommendWriteActivity.RATE;
+
+public class DetailFragment extends Fragment {
+
+    private TextView thumbUp, thumbDown, ratingText, writeBtn, movieName, bookRate;
+    private CheckBox thumbUpBtn, thumbDownBtn;
+    private RatingBar ratingbar;
+    private Button showAllBtn;
+    private ReviewAdapter adapter;
+    private ArrayList<ReviewitemData> items;
+    private ListView reviewListView;
+    private ScrollView scrollview;
+    private ImageView movieImage;
+
+    private MainActivity activity;
+
+    public final static String ITEMS  = "items";
+    private final static int WRITE_REQ_CODE = 101;
+    private final static int SHOW_REQ_CODE = 20;
+
+    @Override
+    public void onAttach(@NonNull Context context) {
+        super.onAttach(context);
+        activity = (MainActivity)getActivity();
+    }
+
+    @Nullable
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
+        View root = inflater.inflate(R.layout.fragment_detail, container, false);
+
+        thumbUp = root.findViewById(R.id.thumb_up_text);
+        thumbDown = root.findViewById(R.id.thumb_down_text);
+        thumbUpBtn = root.findViewById(R.id.thumb_up_btn);
+        thumbDownBtn = root.findViewById(R.id.thumb_down_btn);
+        ratingbar = root.findViewById(R.id.rating_bar);
+        ratingText = root.findViewById(R.id.rating_text);
+        writeBtn = root.findViewById(R.id.write_btn);
+        showAllBtn = root.findViewById(R.id.show_all_btn);
+        reviewListView = root.findViewById(R.id.review_listview);
+        scrollview = root.findViewById(R.id.scrollview);
+        movieImage = root.findViewById(R.id.movie_image);
+        movieName = root.findViewById(R.id.movie_name);
+        bookRate = root.findViewById(R.id.book_rate_text);
```

Comment:
초기화 하는 코드는 한군데 묶어 함수로 빼주는 것도 좋아 보입니다.
```suggestion
        initView() or setupView()
```